tests/nodes/supporting_nodes/test_group.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+import cript
+from cript.nodes.exceptions import UneditableAttributeError
+
+
+def test_create_group_from_json():
+    """
+    simply tests if a valid group node can be created from API JSON
+    """
+    pass
+
+
+@pytest.fixture(scope="session")
+def group_node() -> cript.Group:
+    """
+    create a group from JSON that can be used by other tests
+
+    Notes
+    -----
+    User node should only be created from JSON and not from instantiation
+
+    Returns
+    -------
+    Group
+    """
+    # TODO create group object from JSON instead of code
+    my_group = cript.Group(name="my group", admins=["admin 1"], users=["user 1"])
+    return my_group
+
+
+def test_group_to_json(group_node) -> None:
+    """
+    tests if the group node can be correctly serialized to JSON
+    """
+    pass
```

Comment:
I ended up putting serialization of group and deserialization into the same test function because I cannot instantiate a Group node and compare the group node from json and an instantiated group json and be sure they are the same. Instead 

I am converting a group json to node
then back to json
and checking that it is the same.

I am thinking since the node is read only for now just seeing the json serialization and deserialization work correctly is good.

what do you think?

Reply:
I think that is OK for now.
Not ideal, because it maybe that both just return empty strings for json.
For testing my nodes, I have a reference string that I compare the json against.

Reply:
I think you are right!
I think it's fine for now as well. In the future, we can come back and update it if we need to